verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -35,13 +35,29 @@ ABSL_FLAG(bool, printextraction, false,
           "Whether or not to print the extracted general indexing facts "
           "from the middle layer)");
 
-// TODO(ikr): Rename to print_kythe_facts_json
-ABSL_FLAG(bool, printkythefacts, false,
+ABSL_FLAG(bool, print_kythe_facts_json, false,
           "If true, prints the extracted Kythe facts in JSON format");
 
 ABSL_FLAG(bool, print_kythe_facts_proto, false,
           "If true, prints the extracted Kythe facts in protobuf format");
 
+ABSL_FLAG(
+    std::string, file_list_path, "",
+    R"(The path to the file list which contains the names of SystermVerilog files.)");
```

Comment:
"SystemVerilog"

also: "... the files should be ordered by definition dependencies."

(most people will understand)